superset-frontend/src/pages/DvtHome/index.tsx

```diff
@@ -26,22 +26,162 @@ import {
   DataContainer,
   CalendarContainer,
 } from './dvt-home.module';
+import DvtTitleCardList, {
+  CardDataProps,
+} from 'src/components/DvtTitleCardList';
+
+const cardData: CardDataProps[] = [
+  {
+    id: 1,
+    title: 'Card 1',
+    label: 'Label 1',
+    description: 'Description 1',
+    isFavorite: false,
+    link: '/link1',
+  },
+];
 
 function DvtWelcome() {
   const [openCalendar, setOpenCalendar] = useState<boolean>(true);
   const [calendar, setCalendar] = useState<Moment | null>(null);
+  const [favorites, setFavorites] = useState<number[]>([]);
+  const [recentData, setRecentData] = useState<CardDataProps[]>([]);
+  const [dashboardData, setDashboardData] = useState<CardDataProps[]>([]);
+  const [chartData, setChartData] = useState<CardDataProps[]>([]);
+  const [savedQueriesData, setSavedQueriesData] = useState<CardDataProps[]>([]);
+
+  const handleSetFavorites = (id: number, isFavorite: boolean) => {
+    if (isFavorite) {
+      setFavorites(prevFavorites => [...prevFavorites, id]);
+    } else {
+      setFavorites(prevFavorites =>
+        prevFavorites.filter(favoriteId => favoriteId !== id),
+      );
+    }
+  };
+
+  useEffect(() => {
+    const fetchRecentData = async () => {
+      try {
+        const response = await fetch(`/api/v1/chart/`);
+        const data = await response.json();
+        setRecentData(data);
+      } catch (error) {
+        console.error('API request failed:', error);
+      }
+    };
+
+    const fetchDashboardData = async () => {
```

Comment:
4 tane benzerlik fetch ve for bi tane yeni function callback kullanmalı. (minify için)